_ap/18/index.adoc

```diff
@@ -3,76 +3,49 @@ num: 18
 title: "OpenAPI clients code generation - Kiota"
 status: "Draft"
 authors:
-- "andreaTP"
+- "Andrea Peruffo"
 tags: []
 ---
 
-// Top style tips:
-// * Use one sentence per line
-// * No unexpanded acronyms
-// * No undefined jargon
-
-// You don't have to use the following sections, but they provide a 
-// useful structure for writing a clear document.
-
 ## Intent
-// Summarize in a single sentence what the pattern tries to achieve.
-//
-// For example:
-//
-// Define a policy for exposing metrics to customers in a consistent way.
+
+Define a consistent technology to be used for client code generation from OpenAPI specifications.
 
 ## Motivation
 
-// In a few paragraphs describe the motivating factors for this pattern
-//
-// For example:
-// 
-// Prometheus scrape endpoints are the defacto standard for exposing metric information for collection.
-// However this format and the associated conventions are not sufficient in themselves to ensure
-// that metrics are exposed in a consistent way. Problems include:
-// * Insufficient guidance on metric and label naming to provide consistency and 
-//   establishing the principal of hiding implementation details.
-// * Treatment of this as a first-class API, with established mechanisms for API evolution
-// * etc.
+* Have a uniform and good quality SDK layer to be used for APIs access
```

Comment:
* offer a client or SDK for Services APIs that CLI, UI and customers can rely on

Reply:
The fact that I left this one out was actually intentional ... I can get it back, but my intention was to:

> leave the door open to (eventually)provide only "tooling" and not full-fledged SDKs

I understand this might sound a little excessive at this stage but, at least, I would like to keep this door open.
Thoughts?

Reply:
When you leave the door open, you assume a lot of imagination from a casual reader looking to know if his problem is solved. And that reader ahs to read a lot of things so won't meditate on it. I would go for explicit, you can add tooling as an extra option but the main use case being the sentence you left out.

Reply:
ok, fair 👍

Reply:
added